### ISSUE TITLE
perlfaq5 correction

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#4133]
Migrated from [rt.perl.org#7179](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=7179) (status was 'resolved')

Searchable as RT7179$

**Comment by p5pRT** [comment]
### From @eserte




---
In perlfaq5\.pod\, there are some references to Glob&#8203;::KGlob\. This
should be corrected to File&#8203;::KGlob\, as there is no Glob&#8203;::\* module\.

Regards\,
&nbsp;    Slaven




---


---
Flags&#8203;:
&nbsp;    category=docs
&nbsp;    severity=medium


---
Site configuration information for perl v5\.6\.1&#8203;:

Configured by eserte at Sat Apr 28 16&#8203;:22&#8203;:45 CEST 2001\.

Summary of my perl5 \(revision 5\.0 version 6 subversion 1\) configuration&#8203;:
&nbsp;  Platform&#8203;:
&nbsp;    osname=freebsd\, osvers=3\.5\-stable\, archname=i386\-freebsd\-64int
&nbsp;    uname='freebsd vran\.herceg\.de 3\.5\-stable freebsd 3\.5\-stable \#11&#8203;: sat dec 23 15&#8203;:59&#8203;:48 cet 2000 eserte@&#8203;vran\.herceg\.de&#8203;:usrsrcsyscompilevran i386 '
&nbsp;    config\_args='\-Dprefix=/usr/perl5\.6\.1 \-D usedevel=define \-de \-D hintfile=myfreebsd'
&nbsp;    hint=recommended\, useposix=true\, d\_sigaction=define
&nbsp;    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
&nbsp;    useperlio=undef d\_sfio=undef uselargefiles=define usesocks=undef
&nbsp;    use64bitint=define use64bitall=undef uselongdouble=undef
&nbsp;  Compiler&#8203;:
&nbsp;    cc='cc'\, ccflags ='\-I/usr/local/include'\,
&nbsp;    optimize='\-O2 \-m486'\,
&nbsp;    cppflags='\-I/usr/local/include'
&nbsp;    ccversion=''\, gccversion='2\.7\.2\.3'\, gccosandvers=''
&nbsp;    intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=12345678
&nbsp;    d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=12
&nbsp;    ivtype='long long'\, ivsize=8\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=8
&nbsp;    alignbytes=4\, usemymalloc=n\, prototype=define
&nbsp;  Linker and Libraries&#8203;:
&nbsp;    ld='cc'\, ldflags ='\-Wl\,\-E  \-L/usr/local/lib'
&nbsp;    libpth=/usr/lib /usr/local/lib
&nbsp;    libs=\-lbind \-lgdbm \-lm \-lc \-lcrypt \-liconv \-lutil
&nbsp;    perllibs=\-lbind \-lm \-lc \-lcrypt \-liconv \-lutil
&nbsp;    libc=/usr/lib/libc\.so\, so=so\, useshrplib=false\, libperl=libperl\.a
&nbsp;  Dynamic Linking&#8203;:
&nbsp;    dlsrc=dl\_dlopen\.xs\, dlext=so\, d\_dlsymun=undef\, ccdlflags=' '
&nbsp;    cccdlflags='\-DPIC \-fpic'\, lddlflags='\-shared  \-L/usr/local/lib'

Locally applied patches&#8203;:
&nbsp;    



---
@&#8203;INC for perl v5\.6\.1&#8203;:
&nbsp;    /usr/perl5\.6\.1/lib/5\.6\.1/i386\-freebsd\-64int
&nbsp;    /usr/perl5\.6\.1/lib/5\.6\.1
&nbsp;    /usr/perl5\.6\.1/lib/site\_perl/5\.6\.1/i386\-freebsd\-64int
&nbsp;    /usr/perl5\.6\.1/lib/site\_perl/5\.6\.1
&nbsp;    /usr/perl5\.6\.1/lib/site\_perl
&nbsp;    \.



---
Environment for perl v5\.6\.1&#8203;:
&nbsp;    HOME=/home/e/eserte
&nbsp;    LANG \(unset\)
&nbsp;    LANGUAGE \(unset\)
&nbsp;    LC\_ALL=de\_DE\.ISO\_8859\-1
&nbsp;    LD\_LIBRARY\_PATH \(unset\)
&nbsp;    LOGDIR \(unset\)
&nbsp;    PATH=/usr/X11R6/bin&#8203;:/usr/X11/bin&#8203;:/usr/local/bin&#8203;:/usr/bin&#8203;:/bin&#8203;:/usr/gnu/bin&#8203;:/usr/TeX/bin&#8203;:/usr/local/sbin&#8203;:/usr/sbin&#8203;:/sbin&#8203;:/usr/local/pilot/bin&#8203;:/home/e/eserte/bin/FreeBSD&#8203;:/home/e/eserte/bin/sh&#8203;:/home/e/eserte/bin&#8203;:/usr/X386/bin&#8203;:/usr/games&#8203;:/home/e/eserte/devel
&nbsp;    PERL\_BADLANG \(unset\)
&nbsp;    SHELL=/usr/local/bin/tcsh

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

> In perlfaq5\.pod\, there are some references to Glob&#8203;::KGlob\. This
> should be corrected to File&#8203;::KGlob\, as there is no Glob&#8203;::\* module\.
>

Patch to fix&#8203;:


<details><summary>Inline Patch</summary>

```diff
--- pod/perlfaq5.pod~	Mon Jun 25 14:34:43 2001
+++ pod/perlfaq5.pod	Mon Jun 25 14:36:36 2001
@@ -469,7 +469,7 @@
 Use the <> (glob()) operator, documented in L<perlfunc>.  Older
 versions of Perl require that you have a shell installed that groks
 tildes.  Recent perl versions have this feature built in. The
-Glob::KGlob module (available from CPAN) gives more portable glob
+File::KGlob module (available from CPAN) gives more portable glob
 functionality.

 Within Perl, you may use this directly:
@@ -569,7 +569,7 @@
 have this problem, but their users may be surprised by it.

 To get around this, either upgrade to Perl v5.6.0 or later, do the glob
-yourself with readdir() and patterns, or use a module like Glob::KGlob,
+yourself with readdir() and patterns, or use a module like File::KGlob,
 one that doesn't use the shell to do globbing.

 =head2 Is there a leak/bug in glob()?



/J\
```
</details>

**Comment by p5pRT** [comment]
### From @jhi

Thanks\, applied\.
